src/averageAges.js

```diff
@@ -14,12 +14,11 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const menArray = people.filter(men => men.sex === 'm'
+    && (century ? Math.ceil(men.died / 100) === century : true));
```

Comment:
```suggestion
  const men = people.filter(men => men.sex === 'm'
    && (century 
      ? Math.ceil(men.died / 100) === century 
      : true)
  );
```

Do not use data types in variable names